instill-ai/x
# fix: refactor checkfield functions and add tests

Because

- Test whether the functions works as expected

This commit

- refactor functions in `checkfield`
  - check more types of fields and limitations of the functions
- add `checkfield` tests
